.env-example

```diff
@@ -1,3 +1,5 @@
+APP_ENV=local
```

Comment:
NODE_ENV isn't always what we expect it to be so use a custom var for our environment switcher.